src/social-network-provider/facebook.com/UI/collectPosts.tsx

```diff
@@ -11,12 +11,23 @@ import {
     makeTypedMessageCompound,
 } from '../../../protocols/typed-message'
 import { Flags } from '../../../utils/flags'
+import { clickSeeMore } from './injectPostInspector'
 
 const posts = new LiveSelector().querySelectorAll<HTMLDivElement>(
     isMobileFacebook ? '.story_body_container ' : '[data-testid] [role=article] [data-ad-preview="message"]',
 )
 
 export function collectPostsFacebook(this: SocialNetworkUI) {
+    new MutationObserverWatcher(posts)
```

Comment:
Clicking the expand button to update the text of post node should happen before read its text